cli/cmd/resources.go

```diff
@@ -57,12 +57,13 @@ var (
 
 	envTokensClient = resourcemanager.NewClient(
 		httpClient, cliLogger,
-		"token", "tokens",
+		"token", "environmenttokens",
```

Comment:
shouldn't singular and plural be the same? like token/tokens or environmenttoken/environmenttokens?